### ISSUE TITLE
Should the Crawling spec mention the Discovery TSG more prominently?

### ISSUE THREAD

**Issue by aisaac** [IIIF/discovery#16]
The current draft at http://preview.iiif.io/api/discovery/api/discovery/0.1/ doesn't mention our technical spec group. I'm not sure whether there is an official policy about IIIF Spec (not) refering to their community context, but I think it could be helpful:
- for giving context. In fact refering to the charter and the use cases can be a good way to motivate some choices without writing too much new text as motivations
- for calling to action: rather than just giving a pointer to the mailing list, commenters could be invited to join our group - hopefully resulting in better community engagement. In a way this would extend what @azaroth42 started to do in the 'status warning' when hinting at current issues.
- for attribution: if we want to trigger more groups like ours in the future, it's good that one can see them mentioned in what they helped building.

Basically we could look at how W3C specs do it. See the 'status of this document' at https://www.w3.org/TR/annotation-model/ (selected completely randomly of course ;-) )

**Comment by azaroth42** [comment]
TSGs are relatively new as a construct within IIIF, and not as clearly defined as WGs are in the w3c. They're definitely transitory -- when the spec is done, the group is dissolved, mission accomplished.  So the final spec should not have a prominent link...

... but I agree in the interim that encouraging input and discussion is important. I could see a reference in the status warning at the top, to that end.

When the spec is 1.0, the TSG could go into the Acknowledgements section. (See https://github.com/IIIF/api/issues/1496)

**Comment by aisaac** [comment]
I fully agree that IIIF TSGs are not as crisp as at W3C WGs. But I'm surprised by the point about "they're definitely transitory -- when the spec is done, the group is dissolved, mission accomplished." This is exactly how it happens with W3C spec groups, so this point would be in favour of doing like W3C...
Also, I believe that the charter of groups should be prominently showed. Whatever the nature of TSGs or other groups in the future, a group's charter is published on the IIIF site and this should be considered seriously.

All this said, I believe that what @azaroth42 proposes concretely would answer the concerns of my view, even if it's slightly different than his ;-)

**Comment by zimeon** [comment]
I think recognizing the TSG makes complete sense.

**Comment by azaroth42** [comment]
Now in acks, following the pattern in Prezi3
